File: Algorithms/lab10/Azad.cpp
#include <iostream>
#include <fstream>
#include <vector>
 
using namespace std;
 
int main() {
    ifstream fin("input.txt");
    ofstream fout("output.txt");
    int n, m;
    fin >> n >> m;
    int f, k;
    vector<int> arr(n, 0);
    for (int i = 0; i < m; ++i) {
        fin >> f >> k;
        arr[f-1]++;
        arr[k-1]++;
    }
    for (int i : arr) {
        fout << i << " ";
    }
    fin.close();
    fout.close();
    return 0;
}

File: Algorithms/lab8/Azad.py
r = open("input.txt", "r")
n, m = map(int, r.readline().split()) 
arr = [[0] * (n+1) for i in range(n+1)]
for i in range(m):
    f, k = map(int, r.readline().split()) 
    arr[f][k] = 1

g = open("output.txt", "w")
for i in range(1, n+1):
    for j in range(1, n+1):
        g.write(str(arr[i][j]) + " ")
    g.write(str('\n'))
g.close()

    
